server/etc/pulp/server.conf

```diff
@@ -178,6 +178,39 @@
 # task_result_history: 3
 
 
+# = Lazy Loading =
```

Comment:
I thought I already added the `[lazy]` section.  How is this not a merge conflict?